tests/queries/tests.py

```diff
@@ -1172,6 +1172,16 @@ def test_lookup_constraint_fielderror(self):
         with self.assertRaisesMessage(FieldError, msg):
             Tag.objects.filter(unknown_field__name='generic')
 
+    def test_ticket_28297(self):
+        # https://code.djangoproject.com/ticket/28297
```

Comment:
As discussed in [Coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style), use a descriptive docstring. The seems complicated enough that a ticket reference is warranted, however, a reader shouldn't have to look up the ticket to understand the basics of the test.
(sorry forgot to press "submit" on this comment during the first review)